Configuration/Eras/python/Era_Run2_2017_trackingLowPU_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+from Configuration.Eras.Modifier_trackingLowPU_cff import trackingLowPU
+
+Run2_2017_trackingLowPU = cms.ModifierChain(Run2_2017, trackingLowPU)
```

Comment:
if the cleanup was made already, will I still need to change this to run2_2017_core or will this part change as well?
If there is coupling, I'd rather leave this as is.
Matti, please clarify.
Thanks.